compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -323,14 +310,13 @@ class TextInputTests : OnCanvasTests {
             keyEvent("c", type = "keyup"),
         )
 
-        assertEquals("abc", inputInteractor.waitForInput(), "XXX")
+        assertTextEquals("abc", textFieldValue, "XXX")
     }
 
-    fun repeatedAccentMenuClicked() = runTest {
-        val inputInteractor = InputInteractor()
-        inputInteractor.createTextFieldWithChannel()
+    fun repeatedAccentMenuClicked() = runApplicationTest {
```

Comment:
This is not marked with `@Test`. Is it intentional?

Reply:
@ASalavei  asked to turn it off till further investigation - (but it was done not in this PR) - I'll reenable it later on